src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/dynamic-metadata-form/dynamic-metadata-form.component.less

```diff
@@ -6,3 +6,19 @@
 	background-color: @background-color-3;
 	color: white;
 }
+
+.inputs-container {
```

Comment:
place classes in their ancestor class. In general try to keep classes in nested form